docs/_parser.lua

```diff
@@ -190,7 +225,21 @@ local function create_table(entries, columns, prefix)
         local line = "  <tr>"
 
         for _, column in ipairs(columns) do
-            line = line.."<td>"..entry[column].."</td>"
+            if type(entry[column]) == "table" then
+                line = line .. "<td>"
+                local firstline = true
+                for _,v in pairs(entry[column]) do
+                    if not firstline then
+                        line = line .. "<br>"
+                    else
+                        firstline = false
+                    end
+                    line = line .. v
+                end
+                line = line .. "</td>"
```

Comment:
I think it's better to do that with list. This is basically a list that we want to display here, so it feels better to use the right HTML tag for this content, and it makes the code easier to read:

```suggestion
            if type(entry[column]) == "table" then
                line = line .. "<td><ul>"
                for _,v in pairs(entry[column]) do
                    line = line .. "<li>" .. v .. "</li>"
                end
                line = line .. "</ul></td>"
```

If we really need to remove the list's bullets, we can easily do that by CSS (I think we already have a class for that (?)).

Reply:
It's much cleaner, thanks!